effects/effect.py

```diff
@@ -60,13 +55,13 @@ def _validate(self, target_number: int) -> bool:
         pass
 
     def resolve(self) -> None:
-        if self._status == EffectStatus.ACTIVATED:
+        if (self._status == EffectStatus.ACTIVATED):
             is_finished = self._resolve_impl()
 
             if (is_finished):
                 self._status = EffectStatus.FINISHED
 
-        else if self._status == EffectStatus.CREATED:
+        elif (self._status == EffectStatus.CREATED):
```

Comment:
Are brackets necessary?

Reply:
Not necessary. Added for consistency with other code parts

Reply:
I always write without brackets. Now it is approved but we need to speak about one format for brackets, quotes etc. for each other.